code-challenges.js

```diff
@@ -31,38 +55,92 @@ const temp2 = 350
+        return `${num} is at boiling point`
+    } else {
+        return `${num} is below boiling point`
+    }
+}
+
+console.log(checkBoil(temp1)); //42 is below boiling point
+console.log(checkBoil(temp2)); //350 is above boiling point
+console.log(checkBoil(temp3)); //212 is at boiling point
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+//No function is necessary though it is possible to create one that would work.
+//Use the built in concat method to combine the strings then use the built in length method to get the length. 
+//Need to create a new array variable because we are using an accessor method.
+//Also possible to concat with the spread operator like ...arr. This is also an accessor method as it will not change the original array.
+//bonus function that takes in two arrays, concats them, then returns their length
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+let padresCombined = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+console.log(padresCombined.length); // 9
+
+let padsCombined = [...padres1984WorldSeriesRuns, ...padres1998WorldSeriesRuns]
+console.log(padsCombined.length); // 9
+
+let combine = (arr1, arr2) => {
+    let combined = [...arr1, ...arr2]
+    return combined.length
+}
+console.log(combine(padres1984WorldSeriesRuns, padres1998WorldSeriesRuns)); // 9
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
+//Can simply use the built in method .reverse but this will only work on an array.
+//First use .split("") on the string to form it into an array, then .reverse() to reverse the array, then .join("") to form it into a string.
+//Can be used all together within the log.
 
 const currentCohort = "Golf 2022"
 // Expected output: "2202 floG"
+console.log(currentCohort.split("").reverse().join(""));  //2202 floG
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Describe your process:
-
+// Just use the lastIndexOf built in method to log the last index of the given value.
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 const givenValue1 = 42
 // Expected output: 7
 
 const givenValue2 = 10
 // Expected output: 8
 
+console.log(myNumbers.lastIndexOf(givenValue1)); // 7
+console.log(myNumbers.lastIndexOf(givenValue2)); // 8
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Describe your process:
+// Use the built in sort method .sort((a,b)=> b-a) to sort the array from largest to smallest
+//Could also just use the .sort() which will go smallest to largest then reverse the array.
```

Comment:
Keep in mind that sort by default can have an unexpected result with numbers.  For example, if using sort on an array like [1, 25, 113, 75, 9] without the added argument you mentioned on line 124, you would get back [ 1, 113, 25, 75, 9 ].